mercadopago/resources/order.py

```diff
@@ -33,3 +32,131 @@ def create(self, order_object, request_options=None):
+        Args:
+            order_id (str): Order ID
+            request_options (mercadopago.config.request_options, optional): An instance of
+            RequestOptions can be pass changing or adding custom options to ur REST call.
+            Defaults to None.
+
+        Raises:
+            ValueError: Param order_id must be a string
+
+        Returns:
+            dict: Order cancellation response
+        """
+        if not isinstance(order_id, str):
+            raise ValueError("Param order_id must be a string")
+
+        return self._post(uri="/v1/orders/" + str(order_id) + "/cancel", request_options=request_options)
+
+    def capture(self, order_id, request_options=None):
+        """[Click here for more info](https://www.mercadopago.com.br/developers/pt/reference/order/online-payments/capture/post)  # pylint: disable=line-too-long
+        Args:
+            order_id (str): ID of the order to be captured. This value is returned in the response to the Create order request.
+            request_options (mercadopago.config.request_options, optional): An instance of
+            RequestOptions can be pass changing or adding custom options to ur REST call.
+            Defaults to None.
+        Raises:
+            ValueError: Param order_id must be a string
+        Returns:
+            dict: Order ID returned in the response to the request made for its creation.
+        """
+
+        if not isinstance(order_id, str):
+            raise ValueError("Param order_id must be a string")
+
+        return self._post(uri="/v1/orders/" + str(order_id) + "/capture", request_options=request_options)
+
+    def create_transaction(self, order_id, transaction_object, request_options=None):
+        """[Click here for more info](https://www.mercadopago.com.br/developers/pt/reference/order/online-payments/add-transaction/post)  # pylint: disable=line-too-long
+
+        Args:
+            order_id (str): The ID of the order to which the transaction will be added
+            transaction_object (dict): Transaction to be added
+            request_options (mercadopago.config.request_options, optional): An instance of
+            RequestOptions can be pass changing or adding custom options to ur REST call.
+            Defaults to None.
+
+        Raises:
+            ValueError: Param transaction_object must be a Dictionary
+
+        Returns:
+            dict: Transaction addition response
+        """
+        if not isinstance(transaction_object, dict):
+            raise ValueError("Param transaction_object must be a Dictionary")
+
+        response = self._post(uri=f"/v1/orders/{order_id}/transactions", data=transaction_object,
+                              request_options=request_options)
+        if response.get("status") != 201:
+            raise Exception(f"Failed to add transaction: {response}")
```

Comment:
[nth] Como o nome do método está com `create`, sugiro trocar a mensagem de erro também.
```suggestion
            raise Exception(f"Failed to create transaction: {response}")
```